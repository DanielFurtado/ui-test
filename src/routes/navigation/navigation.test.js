import renderer from 'react-test-renderer';
import {BrowserRouter as Router} from 'react-router-dom';

import Navigation from './navigation.component';

describe(('Navigation'), () => {
  it('renders', () => {
    const component = renderer.create(
      <Router>
        <Navigation />
      </Router>
      );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

