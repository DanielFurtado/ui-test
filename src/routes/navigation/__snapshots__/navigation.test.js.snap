// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders 1`] = `
<header
  className="navigation"
  role="banner"
>
  <nav
    className="nav-container"
  >
    <a
      aria-current="page"
      className="logo-container active"
      href="/"
      onClick={[Function]}
    >
      <svg
        className="logo"
      >
        motorway-logo.svg
      </svg>
    </a>
    <ul
      className="nav-links-container"
      role="menubar"
    >
      <li
        role="none"
      >
        <a
          aria-current="page"
          className="nav-link active"
          href="/"
          onClick={[Function]}
          role="menuitem"
        >
          Gallery
        </a>
      </li>
      <li
        role="none"
      >
        <a
          className="nav-link"
          href="/sign-up"
          onClick={[Function]}
          role="menuitem"
        >
          Sign up
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
